e2e-tests/tfm/cypress/e2e/journeys/case-amendments/6. cover-end-date.spec.js

```diff
@@ -108,79 +106,52 @@ context('Amendments - Cover End Date', () => {
-      cy.login(PIM_USER_1);
-      const facilityId = dealFacilities[0]._id;
-      cy.visit(relative(`/case/${dealId}/facility/${facilityId}`));
-
-      facilityPage.facilityTabAmendments().click();
-      amendmentsPage.continueAmendmentButton().click();
-      cy.url().should('contain', 'request-date');
-      amendmentsPage.continueAmendment().click();
-      cy.url().should('contain', 'request-approval');
-      amendmentsPage.continueAmendment().click();
-      cy.url().should('contain', 'amendment-options');
-      amendmentsPage.continueAmendment().click();
-      cy.url().should('contain', 'cover-end-date');
-      amendmentsPage.amendmentCoverEndDateDayInput().clear().focused().type(dateConstants.todayDay);
-      amendmentsPage.amendmentCoverEndDateMonthInput().clear().focused().type(dateConstants.todayMonth);
-      amendmentsPage.amendmentCoverEndDateYearInput().clear().focused().type(dateConstants.todayYear);
-      amendmentsPage.continueAmendment().click();
-
-      cy.url().should('contain', 'is-using-facility-end-date');
-    });
-  }
-
-  if (!tfmFacilityEndDateEnabled) {
-    it('should continue to the `Check answers` page if the cover end date is valid, only the cover end date is to be changed and the facility end date FF is disabled', () => {
-      cy.login(PIM_USER_1);
-      const facilityId = dealFacilities[0]._id;
-      cy.visit(relative(`/case/${dealId}/facility/${facilityId}`));
-
-      facilityPage.facilityTabAmendments().click();
-      amendmentsPage.continueAmendmentButton().click();
-      cy.url().should('contain', 'request-date');
-      amendmentsPage.continueAmendment().click();
-      cy.url().should('contain', 'request-approval');
-      amendmentsPage.continueAmendment().click();
-      cy.url().should('contain', 'amendment-options');
-      amendmentsPage.continueAmendment().click();
-      cy.url().should('contain', 'cover-end-date');
-      amendmentsPage.amendmentCoverEndDateDayInput().clear().focused().type(dateConstants.todayDay);
-      amendmentsPage.amendmentCoverEndDateMonthInput().clear().focused().type(dateConstants.todayMonth);
-      amendmentsPage.amendmentCoverEndDateYearInput().clear().focused().type(dateConstants.todayYear);
-      amendmentsPage.continueAmendment().click();
-
-      cy.url().should('contain', 'check-answers');
-
-      amendmentsPage.amendmentAnswerBankRequestDate().should('contain', dateConstants.todayDay);
-      amendmentsPage.amendmentAnswerRequireApproval().should('contain', 'Yes');
-      amendmentsPage.amendmentAnswerCoverEndDate().should('contain', dateConstants.todayDay);
-      amendmentsPage.amendmentAnswerIsUsingFacilityEndDate().should('not.exist');
-    });
+  it('should continue to the `Check answers` page if the cover end date is valid and only the cover end date is to be changed', () => {
+    cy.login(PIM_USER_1);
+    const facilityId = dealFacilities[0]._id;
+    cy.visit(relative(`/case/${dealId}/facility/${facilityId}`));
```

Comment:
could the 3 lines be put in a beforeEach where the facilityId is passed? it is repeated twice

Reply:
Yes sure, have amended